ChatDemo/SceneDelegate.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SceneDelegate.swift
+//  ChatDemo
+//
+//  Created by Matheus Cardoso on 6/16/20.
+//  Copyright © 2020 Stream. All rights reserved.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+        guard let _ = (scene as? UIWindowScene) else { return }
```

Comment:
Tutorial has code to reload channelsPresenter here

Reply:
Done 👍